ngraph/src/ngraph/validation_util.hpp

```diff
@@ -89,6 +89,15 @@ namespace ngraph
                                  const PartialShape& gamma_shape,
                                  const PartialShape& beta_shape);
 
+    NGRAPH_API
+    bool try_apply_infer_auto_padding(const PartialShape& image_shape,
```

Comment:
There is something wrong with the name of this function. I can't tell what it's supposed to do.
Please change name and add some documentation.